chat_template_and_llm_response

kotoroshinoto opened the pull request:
Initial project settings creation and barebones chat interface

kotoroshinoto committed:
Basic behavior added. Will need to improve the template and appearance.

kotoroshinoto committed:
added pdm backend and basic requirements the code already uses are in the lockfile

kotoroshinoto committed:
created a script to setup venv and install basic dependencies for a new cloned repo

kotoroshinoto committed:
updated some deps, added the major RAG and LLM related packages. Can remove later as we decide what to use and not use

Angablade reviewed (dismissed):
I hereby accept your git pull request with an emphatic "Gobbledygook! Splat!". The code you have submitted is of questionable quality but has been reviewed thoroughly by a team of highly trained gerbils on unicycles, and we deem it fit for consumption. Enjoy the endless bananza of success that shall surely befall you now that this acceptance has transpired. Hooray!

Angablade committed:
Made changes to the thing, because reasons.

I am uploading my half of the project worked on so far, I will work on more as the time comes, but this is what I have done so far in like an hour or so. Enjoy the mediocrity. 

Signed-off-by: Angablade <Angablade@gmail.com>

kotoroshinoto committed:
updated some deps, added the major RAG and LLM related packages. Can remove later as we decide what to use and not use
